src/base64/base64.test.ts

```diff
@@ -5,9 +5,13 @@
  */
 
 import { describe } from "vitest";
-import { compressToBase64, decompressFromBase64 } from ".";
-import { runTestSet } from "../__tests__/testFunctions";
+import { compressToBase64, compressToBase64_fixed } from ".";
+import { decompressFromBase64, decompressFromBase64_fixed } from ".";
```

Comment:
Should merge into one import (format might want to play though!)

```suggestion
import {
    compressToBase64,
    compressToBase64_fixed,
    decompressFromBase64,
    decompressFromBase64_fixed,
} from ".";
```